lib/internal/child_process.js

```diff
@@ -526,6 +528,7 @@ class Control extends EventEmitter {
   constructor(channel) {
     super();
     this.#channel = channel;
+    this[kPendingMessages] = new SafeSet();
```

Comment:
Is there a specific advantage of using a set over a plain array here?

Reply:
it's not. I put it there because I prefer the API to be more readable than an array. Is there a disadvantage to it? I assume that we'd have a few pending messages for this situation idk if for performance it'd be bad

Reply:
Yeah, as you’re mentioning, the disadvantage is performance :) And also semantics, `Set`s specifically are a data structure for eliminating duplicates, so using a `Set` might give the impression that that matters here even though it doesn’t

Reply:
Perfect, thank you! Just sent the commit with the fix o/